### PR TITLE
CI and 3rdparty updates

### PR DESCRIPTION
**Pull request by scribam**
- Update AppVeyor / Travis to use a newer Vulkan SDK
- Update VulkanMemoryAllocator to the latest tagged version
- Update stb_image.h / stb_truetype.h files

### CODE REVIEW

**Comment by AniLeo** [comment]
Are you also going to update the docker to xenial so we can update to newer SDK on the AppImage? Otherwise versions will not be the same

**Comment by scribam** [comment]
I don't plan to do it currently.
What versions are you referring to?

**Comment by AniLeo** [comment]
I see you already updated the docker, didn't notice before 👍

**Comment by AniLeo** [comment]
If you can't fix travis just ignore the mac builds state like before, those are not working properly so they can be ignored for now

File: rpcs3/Emu/RSX/VK/VulkanAPI.h [hunk 1]
```diff
@@ -14,21 +14,6 @@
 #include "define_new_memleakdetect.h"
 #include "Utilities/types.h"
 
-// TODO: Remove when packages catch up, ubuntu is stuck at 1.1.73 (bionic) and 1.1.82 (cosmic)
-// Do we still use libvulkan-dev package on travis??????
-#if VK_HEADER_VERSION < 95
```
Comment (kd-11) [inline rpcs3/Emu/RSX/VK/VulkanAPI.h]: Don't remove this, not all builds are done via travis/docker.

Comment (scribam) [reply rpcs3/Emu/RSX/VK/VulkanAPI.h]: The docker image `rpcs3/rpcs3-travis-trusty:1.1` uses `libvulkan1_1.1.106.0` / `libvulkan-dev_1.1.106.0`. Check this PR: https://github.com/hcorion/rpcs3-docker/pull/1

Comment (scribam) [reply rpcs3/Emu/RSX/VK/VulkanAPI.h]: Nvm, I get it. I will follow your advice

Comment (kd-11) [reply rpcs3/Emu/RSX/VK/VulkanAPI.h]: Option 2: Remove this and add instructions in readme about adding the lunarg ppa for ubuntu users. Ubuntu LTS is still at 1.1.73.

Comment (scribam) [reply rpcs3/Emu/RSX/VK/VulkanAPI.h]: Option 1 done.

**Pull request merged**